components/sharedfiles.js

```diff
@@ -0,0 +1,199 @@
+var SteamCommunity = require('../index.js');
+var SteamID = require('steamid');
+
+
+/**
+ * Deletes a comment from a sharedfile's comment section
+ * @param {SteamID | String} userID - ID of the user associated to this sharedfile
+ * @param {String} sid - ID of the sharedfile
+ * @param {String} cid - ID of the comment to delete
+ * @param {function} callback - Takes only an Error object/null as the first argument
+ */
+SteamCommunity.prototype.deleteSharedfileComment = function(userID, sid, cid, callback) {
```

Comment:
`sid` is kinda already known to be short for SteamID, could we call this parameter `sharedFileId` or `fileId` instead?

Reply:
Yup, makes sense. I changed all `sid` occurrences to `sharedFileId`.